src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -543,13 +543,30 @@ public FeedbackResponse getFeedbackResponseEntityOptimized (FeedbackResponseAttr
     }
     
     /**
+     * Updates the feedback response identified by {@code newAttributes.getId()} and 
+     *   changes the {@code updatedAt} timestamp to be the time of update.
+     * For the remaining parameters, the existing value is preserved 
+     *   if the parameter is null (due to 'keep existing' policy).<br> 
+     * Preconditions: <br>
+     * * {@code newAttributes.getId()} is non-null and correspond to an existing feedback response.
+     * @throws EntityDoesNotExistException 
+     * @throws InvalidParametersException 
+     */
+    public void updateFeedbackResponse(FeedbackResponseAttributes newAttributes)
+        throws InvalidParametersException, EntityDoesNotExistException {
```

Comment:
@damithc This is the previous comment (before changes):
<img width="731" alt="screen shot 2016-03-14 at 15 20 50" src="https://cloud.githubusercontent.com/assets/8586215/13737463/777adc1c-e9f8-11e5-9dda-2dbc4ce4b447.png">

I indented the <code>changes the {@code updatedAt}...</code> based on the text <code>if the parameter is null...</code> which is indented that way.

Is there a specific way that you want me to change the indentation?